backend/package.json

```diff
@@ -83,6 +83,12 @@
 			"type": "object",
 			"title": "Application Wizard",
 			"properties": {
+				"ApplicationWizard.enableSapWebAnalytics": {
+					"type": "boolean",
+					"default": true,
```

Comment:
Can BAS developers choose not to report WebAnalytics? (change the default to false)